dev-client/src/components/sites/LocationDashboardScreen.tsx

```diff
@@ -0,0 +1,89 @@
+import {ScreenDefinition, useNavigation} from '../../screens/AppScaffold';
```

Comment:
note: i am starting to make screen files not in the `screens` folder. there are going to be a lot of screens, especially when we get to e.g. data inputs and sub screens, so i think we were gonna have to split them up at some point to avoid that folder getting monstrous

Reply:
i'm not sure if screen groups could be helpful here? i don't really understand what their point is so i'm not sure